### PR TITLE
Optimize output by adding color to help information

### PR DESCRIPTION
**Pull request by Tom-DaLin**
Sorry for using Chinese in the comment, I put the original Chinese comment [here](https://github.com/Tom-DaLin/summer-2021/blob/main/Task1.md).

### CODE REVIEW

File: pkg/util/template.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
+package util
+
```
Comment (fgksgf) [inline pkg/util/template.go:2]: Add the license header, please

**Review by wu-sheng — CHANGES_REQUESTED** [review]
@Tom-DaLin You are submitting a pull request in the Apache official repo. We have to ask you to translate all your comments into English.

File: pkg/util/template.go [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+{{- $plain	:="\x1b[0m" -}}
+`
+
+const AppHelpTemplate = TextFormat + `{{$Green -}}
+NAME:
+	{{$green}}{{.Name}}{{if .Usage}} - {{.Usage}}{{end}}{{$Green}}
+
+USAGE:
+	{{$green}}{{if .UsageText}}{{.UsageText}}{{else}}{{.HelpName}} {{if .VisibleFlags}}[global options]{{end}}{{if .Commands}} command [command options]{{end}} {{if .ArgsUsage}}{{.ArgsUsage}}{{else}}[arguments...]{{end}}{{end}}{{$Yellow}}{{if .Version}}{{if not .HideVersion}}
+VERSION:
```
Comment (fgksgf) [inline pkg/util/template.go:23]: Need to add a blank line before this

File: pkg/util/template.go [hunk 3]
```diff
@@ -0,0 +1,81 @@
+package util
+
+// Variable beginning with a capital letter can make the text bold
```
Comment (fgksgf) [inline pkg/util/template.go:3]:
```suggestion
// TextFormat defines variables of style and colors.
```

**Review by fgksgf — APPROVED** [review]
Tested on macOS and CentOS, and it works well.

Thank you.

**Review by kezhenxu94 — APPROVED** [review]

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**